var/spack/repos/builtin/packages/tau/package.py

```diff
@@ -49,21 +53,76 @@ class Tau(Package):
     variant('download', default=False,
             description='Downloads and builds various dependencies')
     variant('scorep', default=False, description='Activates SCOREP support')
-    variant('openmp', default=True, description='Use OpenMP threads')
-    variant('mpi', default=True,
+    variant(
+        'otf',
+        default=False,
+        description='Activates support of Open Trace Format (OTF)')
+    variant(
+        'binutils',
+        default=False,
+        description='Activates support of BFD GNU Binutils')
+    variant(
+        'libunwind',
+        default=False,
+        description='Activates support of libunwind')
+    variant('likwid', default=False, description='Activates LIKWID support')
+    variant('papi', default=False, description='Activates Performance API')
+    variant('python', default=False, description='Activates Python support')
+    variant('openmp', default=False, description='Use OpenMP threads')
+    variant(
+        'ompt',
+        default=False,
+        description='Activates OMPT instrumentation')
+    variant(
+        'opari',
+        default=False,
+        description='Activates Opari2 instrumentation')
+    variant('mpi', default=False,
             description='Specify use of TAU MPI wrapper library')
-    variant('phase', default=True, description='Generate phase based profiles')
-    variant('comm', default=True,
+    variant(
+        'phase',
+        default=False,
+        description='Generate phase based profiles')
+    variant('comm', default=False,
             description=' Generate profiles with MPI communicator info')
+    variant('shmem', default=False,
+            description='Activates SHMEM support')
+    variant('gasnet', default=False,
+            description='Activates GASNET support')
+    variant('cuda', default=False,
+            description='Activates CUDA support')
+    variant('beacon', default=False, description='Activates BEACON support')
+
+    # Check MPI implementation
+    # try:
+    #  mpirunOut = os.popen('which mpirun')
+    # except OSError as e:
+    #  print "OSError > ", e.errno
+    #  print "OSError > ", e.strerror
```

Comment:
what is the above code for?  If it is not used, please delete it. We don't want any commented code in the repository because it just confuses maintainers.